docs/maya/ribbon.md

```diff
@@ -61,9 +61,23 @@ Follow this steps to create an AndRibbonMuscle deformer:
 | :--- | :--- | :------ | :--------- | :---------- |
 | **Use Custom Stiffness**                  | Boolean | False          | ✓ | Toggles the use of a custom stiffness value. If custom stiffness is used, *Material* and *Stiffness Multiplier* will be disabled and *Stiffness* will be used instead. |
 | **Stiffness**                             | Float   | 10<sup>5</sup> | ✓ | Sets the custom stiffness value. Its value must be greater than 0.0. |
-| **Override Shape Preserve Stiffness** | Boolean | False          | ✓ | Override the shape preservation stiffness with a custom value. If disabled it will use either the material stiffness or the custom stiffness value. |
+| **Override Shape Preserve Stiffness**     | Boolean | False          | ✓ | Override the shape preservation stiffness with a custom value. If disabled it will use either the material stiffness or the custom stiffness value. |
```

Comment:
Can you also please check if there are any other screenshots that need to be updated?

Reply:
I checked all the screenshots that were showing the "Advanced Settings" section, anyway I will double check.

Reply:
There are no other screenshots that need to be updated.